Enable sw rendering.

manxiaoliang opened the pull request:
SwiftShader's GL libraries are getting deprecated in andriod T, and to be replaced with ANGLE on top of SwiftShader's Vulkan library.

Add vulkan.pastel.so in file_contexts

Tracked-On: OAM-104193

feijiang1 commented on graphics/mesa/file_contexts:
```diff
@@ -34,6 +34,7 @@
 /(vendor|system/vendor)/lib(64)?/egl/libEGL_swiftshader\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/egl/libGLESv1_CM_swiftshader\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/egl/libGLESv2_swiftshader\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/vulkan\.pastel\.so u:object_r:same_process_hal_file:s0
```
can remove swiftshader OGL lib setting

manxiaoliang committed:
Enable sw rendering.

SwiftShader's GL libraries are getting deprecated in andriod T,
and to be replaced with ANGLE on top of SwiftShader's Vulkan
library.

Add vulkan.pastel.so in file_contexts and remove swiftshader
EGL lib

Tracked-On: OAM-106619
Signed-off-by: manxiaoliang <xiaoliangx.man@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4879

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4880

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4882

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/4883

Done by "Jiang, Fei" <fei.jiang@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/2830

Done by "Jiang, Fei" <fei.jiang@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by "Jiang, Fei" <fei.jiang@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3226
